libopae++/src/handle.cpp

```diff
@@ -125,6 +109,33 @@ void handle::reset() {
   }
 }
 
+uint32_t handle::read_csr32(uint64_t offset, uint32_t csr_space) const {
+  uint32_t value = 0;
+  if (FPGA_OK == fpgaReadMMIO32(handle_, csr_space, offset, &value)){
```

Comment:
<nit>Do you need a space between )) and {? (applies to the four new fn's)</nit>

Reply:
Probably, but I plan to do a clang-format on this soon so this will get fixed then.